pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -355,18 +373,15 @@ const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, onValueChanged,
                     initialUnit={memorySizeUnit}
                     onValueChange={e => onValueChanged('memorySize', e.target.value)}
                     onUnitChange={value => onValueChanged('memorySizeUnit', value)} />
-                {validationStateMemory === "error" &&
-                <HelpBlock>
-                    <p className="text-danger">{validationFailed.memory}</p>
-                </HelpBlock> }
-                {nodeMaxMemory &&
                 <HelpBlock id="memory-size-helpblock">
-                    {cockpit.format(
+                    {validationStateMemory === "error" && <p>{validationFailed.memory}</p>}
+                    {recommendedMemoryHelpBlock && <p>{recommendedMemoryHelpBlock}</p>}
```

Comment:
I guess you can do something like:
```
let recommendedMemoryHelpBlock = null;
    if (recommendedMemory && recommendedMemory > memorySize) {
        recommendedMemoryHelpBlock = <p>{ cockpit.format(
            "The selected Operating System has recommended memory $0 $1",
            recommendedMemory, memorySizeUnit) }</p>;
    }
...
 <in return()>
{recommendedMemoryHelpBlock}
```
but just detail :) (or maybe I am wrong and in this syntax you cannot assign jsx elements to variable)

Reply:
Done.